amical/tests/test_extraction.py

```diff
@@ -1,42 +1,116 @@
+import astropy
 import munch
+import pytest
 from astropy.io import fits
+from packaging.version import Version
 
 from amical.mf_pipeline.bispect import _add_infos_header
 
 
-def test_add_infos_simulated():
+# Astropy versions for
+ASTROPY_VERSION = Version(astropy.__version__)
+
+
+@pytest.fixture()
+def infos():
+    # SimulatedData avoids requiring extra keys in infos
+    return munch.Munch(orig="SimulatedData", instrument="unknown")
+
+
+@pytest.fixture()
+def commentary_hdr():
+    # Create a fits header with commentary card
+    hdr = fits.Header()
+    hdr["HISTORY"] = "History is a commentary card"
+    return hdr
```

Comment:
btw there is no reason to use fixtures for such small data bits. Here's another advice for writing tests: while it's great to follow the DRY principles for production code, it's actually okay to repeat -small amounts of- code in tests
Using fixtures has a cost on readability/maintainability.
It is worth using them when setting up and tearing down a context is costly, or when it becomes clear that large portions of test code are repeated verbatim. It's not the case here.

Note for disclosure that this is my _opinion_, I don't claim to be a test expert, but I _have_ fallen into common pitfalls in the past so I'm trying to avoid them here.

Reply:
Good to know! Thanks :) I removed both. I kept it for `commentary_infos`, as it is slightly longer, but I can change that as well if needed, just let me know.